samplers/aws/xray/sampling_rule.go

```diff
@@ -0,0 +1,79 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package xray
+
+import "sync"
+
+// ToDo: other fields will be used in business logic for remote sampling
+// centralizedRule represents a centralized sampling rule
+type centralizedRule struct {
```

Comment:
The OTel version won't have centralized and local in the same sampler, so we should remove the word centralized from this codebase

Reply:
I think `centralized` here is not used to differentiate the `local` and `centralized` sampler. Since the rules are retrieved from remote centralized location I thought it would make sense to use `centralizedRule`.

Reply:
If you have an adjective like centralized, you would expect to have a differen tadjective too like non-centralized, otherwise it adds some noise and some confusion. These are all just rules, reservoirs used by the xray sampler so we should be better off removing those